src/history_behaviour.rs

```diff
@@ -64,11 +64,12 @@ pub fn show_history(bot_data: &mut crate::BotData, order: String) {
                             .expect("Movie did not have a watched_or_removed_timestamp in show_history")
                         )
             );
-            
+            if eq(reverse,true) {
```

Comment:
```suggestion
            if reverse {
```

Why compare to true if boolean only has two options?

Reply:
was not aware that rust can do that too 👍🏼